nansat/vrt.py

```diff
@@ -1496,12 +1494,7 @@ def transform_points(self, col_vector, row_vector, dst2src=0,
         lonlat = transformer.TransformPoints(dst2src, xy)[0]
 
         # convert return to lon,lat vectors
-        lonlat = np.array(lonlat)
-        if lonlat.shape[0] > 0:
-            lon_vector = lonlat[:, 0]
-            lat_vector = lonlat[:, 1]
-        else:
-            lon_vector, lat_vector = [], []
```

Comment:
Won't the method be able to return an empty lons and lats? If you could add more information to the previous question (line 1496) it would be more clear

Reply:
Fixed in 96fbd90